### PR TITLE
[WIP] Refactoring Cozy-UI

### PR DESCRIPTION
**Pull request by GoOz**
- [x] Refactoring Buttons
- [x]  Refactoring Layouts
- [x]  Refactoring Nav
- [x]  Refactoring Dialog
- [x]  Refactoring Forms
- [x] Clean up all the remaining files
- [x] Find a way to deal with z-indexes way more easily
- [x] Fix Webpack CSS animation-name hash (@m4dz)

### CODE REVIEW

**Comment by kosssi** [comment]
@GoOz what do you do with this PR?

**Comment by GoOz** [comment]
@kosssi waiting for a bug fix on webpack from @m4dz :/

**Comment by GoOz** [comment]
@m4dz You're very welcome to review this big PR. I 💝 you.

File: stylus/ui-components/button.styl [hunk 1]
```diff
@@ -1,39 +1,115 @@
...
+    padding          1em em(15px, 14px) em(10px, 14px)
     vertical-align   top
-    margin           0 em(8px)
-    border           1px solid grey-09
-    height           em(43px)
-    padding          em(11px) em(15px) em(10px)
-    background-color white
-    font-size(14px)
+    text-align       center
+    font-size        14px /* [1] */
```
Comment (m4dz) [inline stylus/ui-components/button.styl]: > /* [1] */

:heart:

**Comment by m4dz** [comment]
Just :cupid: ! Thanks a lot :rocket:

**Pull request merged**